packages/did-auth-siop-op-authenticator/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@sphereon/ssi-sdk-did-auth-siop-authenticator",
+  "version": "0.0.1",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "veramo": {
+    "pluginInterfaces": {
+      "IDidAuthSiopOpAuthenticator": "./src/types/IDidAuthSiopOpAuthenticator.ts"
+    }
+  },
+  "scripts": {
+    "build": "tsc --build",
+    "generate-plugin-schema": "yarn veramo dev generate-plugin-schema"
+  },
+  "dependencies": {
+    "@sphereon/did-auth-siop": "^0.2.2",
+    "@sphereon/pe-js": "^0.5.0",
+    "@veramo/core": "^3.1.0",
+    "cross-fetch": "^3.1.4"
+  },
+  "devDependencies": {
+    "@veramo/cli": "^3.1.0",
+    "nock": "^13.2.1",
+    "typescript": "^4.5.2"
+  },
+  "files": [
+    "dist/**/*",
+    "src/**/*",
+    "README.md",
+    "plugin.schema.json",
+    "LICENSE"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "git@github.com:Sphereon-Opensource/SSI-SDK.git",
+  "author": "Sphereon <dev@sphereon.com>",
+  "license": "Apache-2.0",
+  "keywords": [
+    "Sphereon",
+    "SSI",
+    "Veramo",
+    "DID",
+    "SIOP",
```

Comment:
Add: SIOPv2, Presentation Exchange, OIDC4VP, OpenID Connect

Reply:
added, also kept SIOP, not sure if you wanted SIOPv2 instead of SIOP, but now they are both in there